submit.go

```diff
@@ -541,7 +570,7 @@ func buildReportTitle(p parsedPayload) string {
 	return trimmedUserText
 }
 
-func buildReportBody(p parsedPayload, quoteChar string) *bytes.Buffer {
+func buildReportBody(p parsedPayload, newline, quoteChar string) *bytes.Buffer {
```

Comment:
this is getting a bit confusing. A comment explaining what exactly the parameters do would be helpful.

Reply:
I'm also not entirely convinced this needs to be different between email and github/gitlab?

Reply:
Email isn't markdown and I didn't want to accidentally break something with extra spaces at the end of each line, but I guess it might be fine to add the spaces for emails too. (GitHub is markdown, but doesn't seem to require the two spaces for newlines. GitLab does require spaces to make a newline)